<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sheet_OperatePage</class>
 <widget class="QWidget" name="Sheet_OperatePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="minimumSize">
          <size>
           <width>601</width>
           <height>401</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>601</width>
           <height>401</height>
          </size>
         </property>
         <widget class="QPushButton" name="pBt_OK">
          <property name="geometry">
           <rect>
            <x>343</x>
            <y>350</y>
            <width>141</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>放置完成</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pBt_ignore">
          <property name="geometry">
           <rect>
            <x>510</x>
            <y>350</y>
            <width>75</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>跳过</string>
          </property>
         </widget>
         <widget class="QTableView" name="tableView_Execute">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>43</y>
            <width>700</width>
            <height>301</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>601</width>
            <height>301</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>601</width>
            <height>301</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="pBt_return">
          <property name="geometry">
           <rect>
            <x>200</x>
            <y>350</y>
            <width>75</width>
            <height>40</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_current">
          <property name="geometry">
           <rect>
            <x>6</x>
            <y>10</y>
            <width>132</width>
            <height>22</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>第   个任务</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_allCount">
          <property name="geometry">
           <rect>
            <x>450</x>
            <y>10</y>
            <width>141</width>
            <height>22</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>共有    个任务</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_title">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>10</y>
            <width>131</width>
            <height>26</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Adobe Arabic</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>取操作</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_error">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>360</y>
            <width>171</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
         <zorder>pBt_OK</zorder>
         <zorder>pBt_ignore</zorder>
         <zorder>tableView_Execute</zorder>
         <zorder>pBt_return</zorder>
         <zorder>label_current</zorder>
         <zorder>label_allCount</zorder>
         <zorder>label_title</zorder>
         <zorder>label_error</zorder>
         <zorder>horizontalSpacer_2</zorder>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
